specification/Metadata/schema/class.property.schema.json

```diff
@@ -0,0 +1,110 @@
+          "minimum": 1,
+          "description": "This property can only be used with a `BLOB` property or an `ARRAY` of `BLOB`. When present, all blob values for this property must be exactly this length in bytes. If the property is an `ARRAY`, then each component must have this byte length. Furthermore, no `offsetBufferView` is needed for specifying the start byte of each blob. When blob In cases where the blob is Base64 encoded in JSON (e.g. as a `defaultValue`), `blobByteLength` describes the **decoded** byte length, not the Base64 string."
+      },
+      "normalized": {
+          "type": "boolean",
+          "description": "Specifies whether integer values are normalized. This applies both when `type` is an integer type, or when `type` is `ARRAY` with a `componentType` that is an integer. For unsigned integer types, values are normalized between [0, 1]. For signed integer types, values are normalized between [-1, 1]. For all other types, this property is ignored.",
+          "default": false
+      },
+      "max": {
+          "type": "array",
+          "description": "Maximum allowed values for property values. Only applicable for fixed-size numeric element types (integer, floats and `ARRAY`s thereof). Both `min` and `max` arrays have the same length. When `type` is `ARRAY` the length is determined by `componentCount`; otherwise the length is 1. `normalized` property has no effect on array values: they always correspond to the integer values.",
+          "items": {
+              "type": "number"
+          },
+          "minItems": 1
+      },
+      "min": {
+          "type": "array",
+          "description": "Minimum allowed values for property values. Only applicable for fixed-size numeric element types (integer, floats and `ARRAY`s thereof). Both `min` and `max` arrays have the same length. When `type` is `ARRAY` the length is determined by `componentCount`; otherwise the length is 1. `normalized` property has no effect on array values: they always correspond to the integer values.",
+          "items": {
+              "type": "number"
+          },
+          "minItems": 1
+      },
+      "default": {
+          "type": ["boolean", "number", "string", "array"],
+          "description": "A default value to use when no value for this property is specified.  This happens only when using JSON encoding and `null` is used to indicate no value. If used, `optional` must be set to `true`. The data type here must match the property declaration. However, `BLOB` properties must be encoded in Base64 encoding since JSON does not support arbitrary binary sequences."
+      },
+      "optional": {
+          "type": "boolean",
+          "description": "If true, this property is optional and can take a null value. This is accomplished by not using a `bufferView` or by using a validity buffer. A validity bit set to 0 indicates the absence of data for that instance. In both cases, `optional` must be set to `true`.",
```

Comment:
Remove mention of validity buffer. Should JSON encoding be mentioned at all?

Reply:
sounds good, and yes, we should mention that this is only usable with JSON encoding.